src/dispatchers.ts

```diff
@@ -235,18 +250,25 @@ export async function followersDispatcher(
     handle: string,
 ) {
     console.log('Followers Dispatcher');
-    const results = (await ctx.data.db.get<string[]>(['followers'])) || [];
-    console.log(results);
-    let items: Person[] = [];
-    for (const result of results) {
-        try {
-            const thing = await lookupPerson(ctx, result);
-            if (thing instanceof Person) {
-                items.push(thing);
-            }
-        } catch (err) {
-            console.log(err);
-        }
+    let items: Actor[] = [];
+    const fullResults = await ctx.data.db.get<any[]>(['followers', 'expanded']);
+    if (fullResults) {
+        items = (await Promise.all(
+            fullResults.map((result): Promise<Actor | null> => {
+                if (result.type === 'Person') {
+                    return Person.fromJsonLd(result)
+                }
+                if (result.type === 'Service') {
+                    return Service.fromJsonLd(result);
+                }
```

Comment:
To make it more robust:

```suggestion
                return Object.fromJsonLd(result).then(object => isActor(object) ? object : null);
```

(You need to import `Object` from `@fedify/fedify` in the above.)

Reply:
Thank you :)